editoast/src/views/infra/routes.rs

```diff
@@ -36,18 +35,30 @@ enum WaypointType {
     BufferStop,
 }
 
+#[derive(Debug, Serialize, Deserialize, PartialEq)]
+struct RoutesResponse {
+    starting: Vec<String>,
+    ending: Vec<String>,
+}
+
+impl RoutesResponse {
+    pub fn new(starting: Vec<String>, ending: Vec<String>) -> Self {
+        Self { starting, ending }
+    }
+}
+
```

Comment:
This `new` is unnecessary since it only does assignment and the struct is defined in the same mod than its usages.